.travis.yml

```diff
@@ -10,9 +10,9 @@ env:
     - DJANGO_VERSION='>=1.10,<1.11' POSTGRES_VERSION=9.5
     - DJANGO_VERSION='>=1.11,<1.12' POSTGRES_VERSION=9.5
```

Comment:
Think these two should be deleted now - as we won't be supporting below Django 2.0